pkg/tool/helmclient/helmclient.go

```diff
@@ -17,20 +17,31 @@ limitations under the License.
 package helmclient
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
-	"errors"
 	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
 	"reflect"
 	"strings"
 
+	cm "github.com/chartmuseum/helm-push/pkg/chartmuseum"
 	hc "github.com/mittwald/go-helm-client"
+	"github.com/pkg/errors"
```

Comment:
This project is no longer under development since December 14, 2020. If not necessary, it is not recommended to continue relying on it.